### ISSUE TITLE
Save Windows Settings

### ISSUE THREAD

**Issue by StephenTH35** [AlexanderPro/SmartSystemMenu#19]
Save Windows Settings

    Hi Alexander, I love "SmartSystemMenu", thank you so much for creating and sharing it. I use it in conjunction with a windows automation app that requires opened windows to be in a certain size and position, so you can facilitate mouse clicks, selections, text insertions, navigation, etc.
    You focus a window, and then use "XY", coordinates to click things in the window, make selections, place the cursor in the proper place to insert text, select text and extract it to a variable to be used later, and so on. Basically, anything that you can do manually in the window or dialog with a mouse or a keyboard.
    You code different automation tasks for each task that you want to perform. For these tasks to be able to run multiple times, the position, size, and other properties of the windows or dialogs must always be the same.
    I use the automation app to automate tons of complicated repetitive tasks. I'm using "SmartSystemMenu", version 1.7.2 right now but will be updating to version 1.7.4 shortly.
    Is it possible to use "SmartSystemMenu", to change several window properties, and then save those changes permanently, for that specific window, so that each time, that specific window opens, it will always have those properties...ego, size, position, always on top, transparency, etc? Even between reboots?
    Maybe it could be a new feature, if it doesn't already exist in version 1.7.4. It would have to really identify the window uniquely, because the "Save As", window or dialog for one app, may be totally different, then the "Save As", window or dialog for another. So you couldn't target the window or dialog by just the "Title Bar", you'd have to use multiple parameters. Of course, when you do want to identify the window or dialog by the text in the "Title Bar", you should be able to use an "Asterisk", to indicate that other text may come before or after the text that you have specified.
    So many apps or dialogs throw in text before, or after, the identifying text, in a "Title Bar". "Mozilla Firefox Browser", will change it's "Title Bar", depending on which tab is selected, when you have multiple tabs open. It will prefix "Mozilla Firefox", with the "Tab Title", and sometimes the "Tab", position. So you would need to identify the "Mozilla Firefox", app to "SmartSystemMenu", like this, "*Mozilla Firefox*". So regardless of which "Tab", is selected, "SmartSystemMenu", still targets the "Mozilla Firefox", main window, as the window to set the saved properties to.
    It would be cool if you could target the window or dialog by several parameters such as "Text in the Title Bar", or the "Program", that spawned the window or dialog. Any ways to specifically target the right window or dialog so the correct saved properties from "SmartSystemMenu", are applied.
    Basically, is it possible to select properties for a window or dialog, using "SmartSystemMenu", and then save them permanently, so they're applied each time that window is spawned? Beyond reboots?
    That's my basic question. The rest is a feature request, with explanation. I apologize for the long-windedness.
                                                            Sincerely, Stephen Harris
                                                            Thanks Again, Alexander

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
as seen windows matching by pattern and windows classes and or process names/path by other Software:
![image](https://user-images.githubusercontent.com/46166740/101240712-57069700-36f1-11eb-9338-7ca38746663f.png)

**Comment by AlexanderPro** [comment]
Duplicate of #61
